csharp/ql/lib/ext/generated/System.Reflection.Emit.model.yml

```diff
@@ -172,7 +172,6 @@ extensions:
       - ["System.Reflection.Emit", "ConstructorBuilder", "GetILGenerator", "()", "summary", "df-generated"]
       - ["System.Reflection.Emit", "ConstructorBuilder", "GetILGenerator", "(System.Int32)", "summary", "df-generated"]
       - ["System.Reflection.Emit", "ConstructorBuilder", "GetMethodImplementationFlags", "()", "summary", "df-generated"]
-      - ["System.Reflection.Emit", "ConstructorBuilder", "Invoke", "(System.Object,System.Reflection.BindingFlags,System.Reflection.Binder,System.Object[],System.Globalization.CultureInfo)", "summary", "df-generated"]
```

Comment:
Are the removals in this file expected?

Reply:
Yes, these removals are of *neutrals* and we don't generate a neutral since we are generating a model for a base implementation of this method
```
 - ["System.Reflection", "MethodBase", True, "Invoke", "(System.Object,System.Reflection.BindingFlags,System.Reflection.Binder,System.Object[],System.Globalization.CultureInfo)", "", "Argument[0]", "Argument[this]", "taint", "df-generated"]
```